src/segment-loader.js

```diff
@@ -637,9 +637,9 @@ export default class SegmentLoader extends videojs.EventTarget {
     }
 
     if (this.loaderType_ === 'main') {
-      const { hasAudio, hasVideo } = this.startingMedia_;
+      const { hasAudio, hasVideo, isMuxed } = this.startingMedia_;
 
-      if (hasVideo && hasAudio && !this.audioDisabled_) {
+      if (hasVideo && hasAudio && !this.audioDisabled_ && !isMuxed) {
```

Comment:
Should this check for the opposite? Since if the track has audio and video and audio is not disabled and isMuxed then we should be using the combined buffered.

Reply:
So in the case of `isMuxed` we will only ever have a `videoBuffer` so we only want to check `videoBuffered`. We could forgo this check here and rely on `sourceUpdater_`. Since `sourceUpdater` already checks what buffers exist before trying to called buffered on them.

Reply:
If we have alt audio available but muxed at first (an option of demuxed), there will be both audio and video buffers (e.g., the first bipbop in the index page).

Reply:
Yeah so we will have to check `isMuxed` here, as in the case of `isMuxed` we only care about the `videoBuffer`.

Reply:
Ah, so even if there are both audio and video buffers, but the stream is muxed and we're not using an alt audio playlist, then we are only appending to the video buffer, but appending both audio and video? Is there a valid case then for ever using the buffered intersection in a segment loader then?

Reply:
Thats a good question and the answer might actually be no, maybe a TODO/issue that we need to open.

Reply:
We only set isMuxed when we have a mixed fmp4/ts playlist where the fmp4 part is muxed.  In that case we *have* to mux together video and audio to get it to keep playing.
Normally for muxed content we split it into separate audio and video in the muxer and append to different buffers. In the normal case we will hit these lines.

Reply:
Ah, so the isMuxed is from the result of the transmuxer, telling us whether we've muxed the content together or not. Got it, thanks!